src/utils/legacyUtil.tsx

```
@@ -59,23 +64,23 @@ export function fillAdditionalInfo(
   function generateMap() {
-    function dig(list: DefaultOptionType[], level = '0', parentIncluded = false) {
+    function dig(list: DataNode[], level = '0', parentIncluded = false) {
       return list
         .map((option, index) => {
           const pos = `${level}-${index}`;
           const value = option[fieldNames.value];
           const included = checkedValues.includes(value);
           const children = dig(option[fieldNames.children] || [], pos, included);
           const node = (
-            <TreeNode {...(option as Required<DefaultOptionType>)}>
+            <TreeNode {...(option as Required<DataNode>)}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)